src/constants/index.ts

```diff
@@ -3,6 +3,16 @@ import { networkConfig } from "./network/";
 const constants = {
   network: { ...networkConfig },
   ALLOWED_POOL_TOKENS: ["WETH", "DAI", "USDC"],
```

Comment:
every time you add a new token, you will have to add it in ALLOWED_POOL_TOKENS as well. Constant ALLOWED_POOL_TOKENS it is in fact Object.values(TOKEN_SYMBOLS_TO_ADDRESS), right?. Do you think it would be better to just use Object.values(TOKEN_SYMBOLS_TO_ADDRESS) everywhere instead of having ALLOWED_POOL_TOKENS  constant?
Or you can do something like that:
```suggestion
  export NETWORK = { ...networkConfig };
  export TOKEN_SYMBOLS_TO_ADDRESS = {
    WETH: "0xc02aaa39b223fe8d0a0e5c4f27ead9083c756cc2",
    DAI: "0x6b175474e89094c44da98b954eedeac495271d0f",
    USDC: "0xa0b86991c6218b36c1d19d4a2e9eb0ce3606eb48",
  };
  export ALLOWED_POOL_TOKENS = Object.keys(TOKEN_SYMBOLS_TO_ADDRESS );
```

Reply:
Done 👍